packages/nx/src/nx-cloud/generators/connect-to-nx-cloud/connect-to-nx-cloud.ts

```diff
@@ -72,20 +73,55 @@ async function createNxCloudWorkspace(
   return response.data;
 }
 
-function printSuccessMessage(url: string) {
-  let origin = 'https://nx.app';
-  try {
-    origin = new URL(url).origin;
-  } catch (e) {}
+function printSuccessMessage(
+  url: string,
+  token: string,
+  installationSource: string
+) {
+  if (process.env.NX_NEW_CLOUD_ONBOARDING !== 'true') {
```

Comment:
What's the plan to remove this?

Reply:
I have no idea!

Reply:
I assume when we are ready to release the feature?